### ISSUE TITLE
qtl update log: +genetic_map column in qtl.tsv and qtlmrk.tsv

### ISSUE THREAD

**Issue by sammyjava** [legumeinfo/datastore-issues#137]
- [x] Arachis/hypogaea/qtl
- [x] Glycine/max/qtl
- [x] Phaseolus/vulgaris/qtl
- [x] Vigna/unguiculata/qtl

**Comment by sammyjava** [comment]
FYI @svengato keeping track here, I'm a big fan of GitHub "tasks" these days.

**Comment by svengato** [comment]
There are two Glycine/max/qtl, one checked and one not.

**Comment by sammyjava** [comment]
Sorry cut and paste error.

NOTE: Arachis/hypogaea/qtl has NEW collections that are just cmap files, not yet converted to map+qtl collections. But the conformant collections are updated. I'll deal with the cmaps when I get back to ArachisMine.

**Comment by sammyjava** [comment]
I'll leave this open in case you find something amiss in the near term, @svengato .

All but one of the vigun "qtl" collections is just qtl-marker associations. It's really GWAS but the p-values weren't reported, which is required for GWAS. So it's in the qtl section. No map, no linkage group, no qtl.tsv file. Fine for you, but I've got an extra tab in the qtlmrk.tsv file to represent the missing genetic_map.

**Comment by svengato** [comment]
It required only a trivial change, I am testing it now.

**Comment by svengato** [comment]
I found some issues:

phavu.Xana_x_Cornell49242.qtl.Murube_Campa_2020.qtlmrk.tsv
Some lines are missing the genetic_map field, and there is only one tab before the marker field, so the marker ends up in the genetic_map column and the distinction ends up in the marker column. This should be easy to fix, just add a tab.

In all of the glyma qtlmrk files, marker is in column 3 and genetic_map in column 4, instead of the other way around (according to the specifications).

glyma.A81356022_x_A81356022.qtl.Keim_Diers_1990.qtlmrk.tsv
"The archive is empty", "No lines available in input"

glyma.M82806_x_HHP.qtl.Brummer_Graef_1997.qtlmrk.tsv
"More columns than column names", some of the lines are missing their newline at the end, so the next line gets appended.

glyma.mixed.qtl.Hwang_King_2016.qtlmrk.tsv
"The archive is empty", "No lines available in input"

glyma.PI97100_x_Coker237.qtl.Mian_Shipe_1997.qtlmrk.tsv
"More columns than column names", some of the lines are missing their newline at the end, so the next line gets appended.

glyma.Young_x_PI416937.qtl.Lee_Bailey_1996b.qtlmrk.tsv
"More columns than column names", some of the lines are missing their newline at the end, so the next line gets appended.

**Comment by sammyjava** [comment]
I think I got it all straightened out. There were a few qtlmrk files created by my script from non-existing files wihch were empty of content so I just deleted those. **ALSO** I had linkage_group in the glyma qtlmrk files, which is incorrect, the marker positions on LGs are in the maps. So I dropped that column. The fifth column, if it exists, is for marker distinction like "Flanking" which we don't have from SoyBase.
